recipes/tex-fmt/recipe.yaml

```
@@ -0,0 +1,42 @@
+context:
+  version: "0.5.2"
+
+package:
+  name: tex-fmt
+  version: ${{ version }}
+
+source:
+  url: https://github.com/WGUNDERWOOD/tex-fmt/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: e9dd89236781ea6f781b00d9452a071311cebd175424baea08bc03e8863f314c
+
+build:
+  number: 0
+  script:
+      - cargo-bundle-licenses --format yaml --output ${{ "%SRC_DIR%" if win else "$SRC_DIR" }}/THIRDPARTY.yml
+      - ${{ "export LD_LIBRARY_PATH=$PREFIX/lib" if unix }}
+      - cargo install --locked --bins --root ${{ "%PREFIX%" if win else "$PREFIX" }} --path .
```

```suggestion
      - cargo install --no-track --locked --bins --root ${{ "%LIBRARY_PREFIX%" if win else "$PREFIX" }} --path .
```
For Windows we usually want the root to be `%LIBRARY_PREFIX%` rather than `%PREFIX%`.  We also want `--no-track` so that some tracking files are cleaned up.